KeyboardSupport.xcodeproj/project.pbxproj

```diff
@@ -636,7 +636,7 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				INFOPLIST_FILE = "Sources/Supporting Files/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
Is this necessary for some piece of functionality in the iOS 11+ SDK? I would rather us leave the deployment target alone unless there is a good reason to bump it up.

Reply:
I think it's the use of `adjustedContentInset` instead of `contentInset`, but I'm not sure if that's required as part of the fix - or if just moving the setting of the variable to the first keyboard observing would do it.

Reply:
Thanks, @wmcginty I didn't notice the use of `adjustedContentInset`, but that does seem to be iOS 11+ only. @Specialist17 was it intentional to use `adjustedContentInset` or was the moving of the code and the `nil` check the main fix you were trying to do here?
```swift
if self?.keyboardScrollableScrollView?.originalContentInset == nil {
    self?.keyboardScrollableScrollView?.originalContentInset = self?.keyboardScrollableScrollView?.contentInset
}
```

Reply:
@tylermilner the moving of the code and the nil check was the main idea here. I'll revert the deployment target change